api.raml

```diff
@@ -273,6 +273,22 @@ schemas:
     responses:
       200:
         description: The new user record was created.
+        body:
+          application/json:
+            description: The newly-created user record.
+            schema: User
+            example: |
+              {
+                "employee_id": "12345",
+                "first_name": "John",
+                "last_name": "Smith",
+                "display_name": "John Smith",
```

Comment:
@fillup Will we generate a `display_name` (either on user creation or dynamically on returning a user's info) even if none was given?